lib/client/keystore.go

```diff
@@ -223,9 +223,27 @@ func (fs *FSLocalKeyStore) DeleteUserCerts(idx KeyIndex, opts ...CertOption) err
 
 // DeleteKeys removes all session keys.
 func (fs *FSLocalKeyStore) DeleteKeys() error {
-	if err := os.RemoveAll(fs.KeyDir); err != nil {
+	const dontDeleteTshConfigFileName = "config"
```

Comment:
Instead of redefining this as a local constant, the configuration file name should be a global constant somewhere so it can be used from multiple places.